src/sections/Company/News-grid/NewsGrid.style.js

```diff
@@ -7,4 +7,36 @@ export const NewsPageWrapper = styled.div`
     .page-header{
         margin:5rem auto;
     }
+    .button-container{
+        display: flex;
+        width: 100%;
+        justify-content: start;
+        margin-bottom: 3rem;
+    }
+    .press-release-button{
+        margin-left: 3rem;
+        background-color: #1E2117;
+        border-radius: 5px;
+        color: white;
+    }   
+    .coverage-button {
+        background-color: #1E2117;
+        border-radius: 5px;
+        color: white;
+    }
+    .filter-topic-input {
+    height: 50px;
+    background-image: url(https://cdn2.hubspot.net/hubfs/4004166/bioticresearch_website_assets/images/search_icon.png);
```

Comment:
let's use an SVG here? referenced from assets?

Reply:
Thanks, trying to implement it now. :smile: